plasma/plasma-mvp/morevp-simplified.md

```diff
@@ -0,0 +1,87 @@
+If `tx` is *not* included in the Plasma chain, then `tx` is not canonical if `other_tx` exists.
+In this sense, the shared input is being double-spent in `tx` because it was already used in `other_tx`. 
+`tx` is canonical if a transaction `other_tx` that would make `tx` not canonical by the above rules does not exist.
+
+We say that a transaction is *fully valid* if it's correctly formed, properly signed, and canonical.
+
+If a transaction is correctly formed and signed, but the transaction *is not* canonical, then, as shown above, some input to that transaction must have been double-spent.
+However, it may be the case that not *all* of the inputs to the transaction are double-spent - other than this transaction, these inputs are unspent.
+In this case, we should allow the owners of these inputs to retrieve their funds.
+The third requirement filters out any inputs that have been double-spent. 
+
+If a transaction is correctly formed and signed, and the transaction *is* canonical, then this is a fully valid transaction.
+In this case, the outputs to this transaction should receive the resulting funds, as is expected.
+Here, the third requirement then simply ensures that only *unspent* outputs will be able to withdraw funds from the contract.
+
+This very basic exit game simply ensures that the “true” owners of money within any transaction will be able to exit.
+
+### Priority
+
+Note, however, that the above exit game fails to provide safety if exit priority is still determined by the position of the transaction input or output in the Plasma chain (“output-age priority”). 
+
+To illustrate, imagine the following scenario:
+
+1. Alice sends transaction `tx` to Bob.
+2. The operator creates a block with invalid transaction `inv_tx`, and includes `tx` after `inv_tx`.
+3. Bob starts the exit game with `tx`.
+4. The operator starts the exit game with `inv_tx`.
+
+Because both `tx` and `inv_tx` appear to be properly formed and signed, both Bob and the operator can prove (1).
+Similarly, because none of the inputs to `tx` and `inv_tx` have been spent in some transaction other than `tx` and `inv_tx`, respectively, both can prove (2).
+Finally, because Bob and the operator have not spent their outputs, both can prove (3). 
+
+Obviously, we don’t want the operator’s exit to be processed before Bob’s exit.
+This will unfortunately be the case if we determine priority based on the age of the output.
+To solve this problem, we change introduce a slight change to how exit priority is handled.
+
+In the MoreVP exit game, within an exit on a transaction `t`, we calculate the priority every input or output of `t` based on the position (block number, transaction index, output index) of the youngest input to `t` (“youngest-input priority”).
+
+Now we can very easily see that, as long as Alice’s input was included in the Plasma chain before the invalid transaction, Bob’s exit will be processed first.
```

Comment:
this isn't enough. The reader will ask what if inputs to `inv_tx` were included before inputs to `tx`. This particular notion of an invalid transaction must be defined and it must be explained, why it is the only way the operator can cheat Alice and Bob (how other attempts to cheat are thwarted).